pkgs/top-level/all-packages.nix

```diff
@@ -29028,6 +29028,8 @@ with pkgs;
 
   ant-nebula-theme = callPackage ../data/themes/ant-theme/ant-nebula.nix { };
 
+  antu-icon-theme = callPackage ../data/icons/antu-icon-theme { };
```

Comment:
I believe these would have to go to [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md) now.